src/utils/utils.ts

```diff
@@ -26,6 +35,45 @@ export function dispatchCustomEvents<T extends keyof CustomEvents>(event: T, ...
     document.dispatchEvent(new CustomEvent(CustomEventId, { detail: JSON.stringify([event, x]) }))
 }
 
+/**
+ * paste image to activeElements
+ * @param bytes
+ */
+export async function pasteImageToActiveElements(bytes: Uint8Array) {
+    const textWrapper = document.createElement('script')
+    textWrapper.setAttribute('id', 'm-aaaaaaaa')
+    textWrapper.setAttribute('type', 'text/x-custom-script')
+    textWrapper.innerHTML = bytes.toString()
+    document.body.append(textWrapper)
+
+    const script = document.createElement('script')
+    script.setAttribute('async', 'false')
+    script.innerHTML = `
+        const s = document.body.querySelector('script#m-aaaaaaaa').innerHTML
```

Comment:
🤔🤔 You may extend injected script to archive your purpose

Reply:
> `const s =`

Reduce bytes used? 🤔

Reply:
I think this is better... A one-time generated script run and destroy, and CSP-proof. Works on iOS and twitter too.

Reply:
...and reduce megabytes used.

Reply:
It's not safe to inject a script tag in the non-init time.

Reply:
could you please explain it? I didn't see if it brings more vulnerabilities to our implementation.